src/methods/polygonize.jl

```diff
@@ -121,8 +121,8 @@ function _polygonize(f::Base.Callable, xs::AbstractRange, ys::AbstractRange, A::
     ybounds = first(ys) - yhalf : step(ys) : last(ys) + yhalf
     Tx = eltype(xbounds)
     Ty = eltype(ybounds)
-    xvec = similar(Vector{Tuple{Tx,Tx}}, xs)
-    yvec = similar(Vector{Tuple{Ty,Ty}}, ys)
+    xvec = similar(Vector{Tuple{Tx,Tx}}, axes(xs))
```

Comment:
```suggestion
    xvec = Vector{Vector{Tuple{Tx,Tx}}}(undef, length(axes(xs)))
```

Wont something like this work and skip the complexity of `similar`?